user/zrepl-dsh2dsh/template.py

```diff
@@ -0,0 +1,24 @@
+pkgname = "zrepl-dsh2dsh"
```

Comment:
just call it zrepl

Reply:
unless there is something in original zrepl that makes it worth preferring over the fork and thus warrants having a separate package for it that would conflict, this should be just zrepl